src/app/(home)/page.tsx

```diff
@@ -62,36 +105,63 @@ export default function HomePage() {
+                            <div className="relative z-10 mt-4 flex flex-row items-center text-2xl font-black lg:text-3xl">
+                                <h3 className="">First-Year Perks</h3>
+                                <Image
+                                    src="/images/yellow-star.svg"
+                                    alt="Yellow Star"
+                                    className="ml-4 h-10"
+                                    width={50}
+                                    height={50}
+                                />
+                            </div>
+                            <div className="relative z-10 mt-4 border-2 border-white bg-grey px-4 py-4 md:px-6 md:py-6">
+                                <p className="text-lg md:text-xl">
+                                    Are you a first year student? The Club runs activities at the
+                                    start of the year specifically for you, giving you a chance to
+                                    meet other students, and helping you ease into uni life.
+                                </p>
+                            </div>
+                        </div>
                     </div>
-                </div>
-                <div className="relative z-10 mt-4 border-2 border-white bg-grey px-4 py-4 md:px-6 md:py-6">
-                    <p className="text-lg md:text-xl">
-                        As a member, some of the perks you&apos;ll have access to include computer
-                        science talks and workshops, catered social events, and a wide network of
-                        other computer science students and graduates to learn from and make friends
-                        with.
-                    </p>
-                </div>
-                <div className="relative z-10 mt-4 flex flex-row items-center text-2xl font-black lg:text-3xl">
-                    <h3 className="">First-Year Perks</h3>
-                    <Image
-                        src="/images/yellow-star.svg"
-                        alt="Yellow Star"
-                        className="ml-4 h-10"
-                        width={50}
-                        height={50}
-                    />
-                </div>
-                <div className="relative z-10 mt-4 border-2 border-white bg-grey px-4 py-4 md:px-6 md:py-6">
-                    <p className="text-lg md:text-xl">
-                        Are you a first year student? The Club runs activities at the start of the
-                        year specifically for you, giving you a chance to meet other students, and
-                        helping you ease into uni life.
-                    </p>
+
+                    {/* <div className="relative z-10 mt-12 text-2xl font-black md:flex-row lg:ml-4 lg:mt-24 lg:text-3xl"> */}
+                    {UPCOMING_EVENTS.length > 0 && (
+                        <div className="relative z-10 mt-12 text-2xl font-black md:flex-row lg:ml-4 lg:mt-24 lg:text-3xl">
```

Comment:
Add a bit more horizontal gap between this and the New Members are Always Welcome section by changing `lg:ml-4` to `lg:ml-10`